.github/workflows/run-tests.yml

```
@@ -0,0 +1,68 @@
+name: Run Test Suite
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+    tags:
+        - '**'
+  pull_request:
+    branches:
+      - '**'
```

We really need a YAML formatter.  I don't super care which style, but this is more consistent at least.

```suggestion
on:
  workflow_dispatch:
  push:
    branches:
    - main
    tags:
    - '**'
  pull_request:
    branches:
    - '**'
```